datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -722,6 +732,10 @@ protected synchronized void handleEvent(CoordinatorEvent event) {
           performPartitionMovement((Long) event.getEventMetadata());
           break;
 
+        case LEADER_DO_POST_BECOMING_LEADER:
```

Comment:
It's really hard to come up with a name for this.. not happy with this because it suggests action with DO but doesnt specify the action. I am unable to come up with any better though. Will post if I do.

Reply:
Shall we call this something like : POST_BECOME_LEADER instead? I don't like the existing name at all

Reply:
I completely agree with both of you. @somandal POST_BECOME_LEADER looks incomplete and is not aligning with the other 3 leader specific event names. How about LEADER_DO_POST_ELECTION ?

Reply:
Just an FYI, the event to handle datastream add/drop is also a leader specific event, so I wouldn't read too much into the "LEADER" naming scheme. I also noticed that the names of events talk about the operation performed. Maybe we should name it along those lines as well? LEADER_DO_POST_ELECTION_CLEANUP or something.

Reply:
Just an FYI, the event to handle datastream add/drop is also a leader specific event, so I wouldn't read too much into the "LEADER" naming scheme. I also noticed that the names of events talk about the operation performed. Maybe we should name it along those lines as well? LEADER_DO_POST_ELECTION_CLEANUP or something.

Reply:
How about LEADER_DO_CLEANUP_POST_ELECTION?  It sounds more readable.

Reply:
Sounds great!